src/fr/lasconic/nwc2musicxml/convert/Nwc2MusicXML.java

```diff
@@ -1600,23 +1991,28 @@ public static void main(String[] args) {
 			File out = new File(args[1]);
 			if (in.exists()) {
 				try {
-					FileInputStream finStream = new FileInputStream(in);
-					InputStream inStream = finStream;
+					FileInputStream finStream1 = new FileInputStream(in);
+					InputStream inStream1 = finStream1;
+					FileInputStream finStream2 = new FileInputStream(in);
+					InputStream inStream2 = finStream2;
 					
 					// check for a [NWZ] header, which indicates that this is a 2.75 NWC file
 					// that contains deflated nwctxt 
 					byte[] refNWZ = {'[','N','W','Z',']',0};
 					byte[] hdrNWZ = new byte[6];
 					boolean compressed = false;
-					if ((inStream.read(hdrNWZ) == 6) && Arrays.equals(hdrNWZ,refNWZ)) {
-						inStream = new InflaterInputStream(finStream);
+					if ((inStream1.read(hdrNWZ) == 6) && Arrays.equals(hdrNWZ,refNWZ)) {
+						inStream1 = new InflaterInputStream(finStream1);
+						inStream2 = new InflaterInputStream(finStream2);
```

Comment:
This will not work. We need `inStream2.read(hdrNWZ)` too or inStream2 will not be the same than inStream1 and it will also not be unzippable..